packages/genesys-spark-components/src/components/stable/gux-popover-list/gux-popover-list.tsx

```diff
@@ -74,6 +79,40 @@ export class GuxPopoverList {
   @Event()
   guxdismiss: EventEmitter<void>;
 
+  /**
+   * Fired when the popover is open
+   */
+  @Event()
+  guxopen: EventEmitter<void>;
+
+  /**
+   * Fired when the popover is close
+   */
+  @Event()
+  guxclose: EventEmitter<void>;
+
+  @Watch('isOpen')
```

Comment:
Will we need to check this also when the component first loads if the `is-open` attribute is already set to `true`.

Reply:
I don't think so
The event is for changes.